src/builtins/test.cpp

```diff
@@ -742,6 +764,11 @@ static bool binary_primary_evaluate(test_expressions::token_t token, const wcstr
         case test_string_not_equal: {
             return left != right;
         }
```

Comment:
I guess this means there is not much use for a dedicated function anymore, we can just inline it.